### DISCUSSION TITLE
Comparison to standard API usage

### DISCUSSION THREAD

**Discussion by peternepp** [fabiangeisler/pyshotgrid#5] [Ideas]
To underline the benefits of using this library, it might be useful to compare standard ShotGrid API calls with this library's call in the readme/documentation side by side.

Like PyMEL used to do.

@fabiangeisler What do you think?

**Comment by fabiangeisler** [comment]
Totally agree on that. I would probably keep it out if the README, but link to it somewhere in the documentation. The README is already fairly long and I want to keep it short-ish with just the necessary information to "sell" the library. Feel free to submit a PR! 🙂

**Comment by fabiangeisler** [comment]
I added a [quick comparison for the two libraries](https://fabiangeisler.github.io/pyshotgrid/pyshotgrid_vs_shotgun_api3.html) to the documentation. Definitely room for improvement there, but its a start 🙂.

**Reply by peternepp** [reply]
Many thanks for that. 😄
